index.bs

```diff
@@ -4569,11 +4569,10 @@ The Registry Track</h3>
 	consisting of one or more associated <dfn export lt="registry table">registry tables</dfn>,
 	each table representing an updatable collection
 	of logically independent, consistently-structured <dfn export lt="registry entry">registry entries</dfn>.
-	A registry has three associated components:
+	It consists of:
```

Comment:
The antecedent is unclear. Better to use the noun than the pronoun, here.
```suggestion
	A registry consists of:
```

Reply:
I wondered about that. I thought it might have been clear enough, but I was unsure. Going back to being explicit.